app/service/service.go

```diff
@@ -0,0 +1,75 @@
+package service
+
+import (
+	"log"
+
+	"github.com/TudorHulban/GoLayouter/app/helpers/helpers"
+	"github.com/TudorHulban/GoLayouter/domain/interfaces"
+	"github.com/TudorHulban/GoLayouter/domain/objects"
+)
+
+type Service struct {
+	paths []interfaces.IFileOperations
+}
+
+func NewService(content []string) *Service {
```

Comment:
let us verify input   
maybe the content is empty... what do we do?
let us add some verification and return an error also

Reply:
addressed in 3383f1a94dc26557ce25920ece2bf590e6b6dc84

Reply:
are you sure it is this commit sha?  
I cannot find updates on this file in the commit you specified

Reply:
addressed in d9c081de52e70f86c4ee0af1d15f3af69343fabf